rdbms/models/telephones.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import Column, Integer, Text, ForeignKey
+from sqlalchemy.exc import IntegrityError
+
+from .database import Base
+from .database import session
+from .users import Users
+
+class Telephones(Base):
+
+    __tablename__='telephones'
+
+    id = Column(Integer, nullable=False, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    telephones = Column(Text)
```

Comment:
Use String instead of Text for this. Text reserves sufficient space to store long-form content such as news articles and is overkill here.